docs/en/latest/plugins/skywalking.md

```diff
@@ -26,25 +26,27 @@ title: skywalking
 - [**Name**](#name)
 - [**Attributes**](#attributes)
 - [**How To Enable**](#how-to-enable)
+- [**How to set endpoint**](#how-to-set-endpoint)
 - [**Test Plugin**](#test-plugin)
-  - [**Run Skywalking Example**](#run-skywalking-example)
+  - [**Running SkyWalking**](#running-skywalking)
 - [**Disable Plugin**](#disable-plugin)
 - [**Upstream services(Code With SpringBoot)**](#Upstream-services(Code-With-SpringBoot))
 
 ## Name
 
-[**Skywalking**](https://github.com/apache/skywalking) uses its native Nginx LUA tracer to provide tracing, topology analysis, and metrics from service and URI perspective.
-The skywalking server can supports both http and grpc protocols. The APISIX client only support http protocols.
+[**SkyWalking**](https://github.com/apache/skywalking) uses its native Nginx LUA tracer to provide tracing, topology analysis, and metrics from service and URI perspective.
+
+The SkyWalking server can support both HTTP and gRPC protocols. Currently, the APISIX client only supports the HTTP protocol.
 
 ## Attributes
 
 | Name         | Type   | Requirement | Default  | Valid        | Description                                                          |
 | ------------ | ------ | ----------- | -------- | ------------ | -------------------------------------------------------------------- |
-| sample_ratio | number | required    | 1        | [0.00001, 1] | the ratio of sample                                                  |
+| sample_ratio | number | required    | 1        | [0.00001, 1] | The ratio of sampling                                               |
```

Comment:
Please use lowercase for the first word like other cells

Reply:
Ok, fixed